bids/tests/utils.py

```diff
@@ -1,7 +1,25 @@
 ''' Test-related utilities '''
+from pathlib import Path
+import pkg_resources
 
-from os.path import join, dirname, abspath
 
+def get_test_data_path(return_type="str"):
```

Comment:
I think we might as well just have this return a `Path` object and update all the tests to either use that, or just wrap with a `str()` in those tests. 

Either way, it's a goal to move over to Pathlib.

We can do this later though, so no worries.